tests/internal/test_utils_http.py

```diff
@@ -0,0 +1,23 @@
+from ddtrace.internal.utils.http import connector
+from tests.utils import MockHTTPServer
+
+
+def test_connector():
+    with MockHTTPServer(port=8181) as server:
```

Comment:
do you want a real server running or would `HTTPretty` be good enough?

Reply:
In theory `HTTPretty` might be enough for this.